frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -42,8 +42,10 @@ Not all NPM packages are fully compatible with all versions of the Console. This
 
 | Console Version | SDK Package | Last Package Version |
 | --- | --- | --- |
-| 4.11.x | `@openshift-console/dynamic-plugin-sdk` | Latest |
+| 4.12.x | `@openshift-console/dynamic-plugin-sdk` | Latest |
 |  | `@openshift-console/dynamic-plugin-sdk-webpack` | Latest |
+| 4.11.x | `@openshift-console/dynamic-plugin-sdk` | 0.0.11 |
```

Comment:
I believe we can use 0.0.12 for 4.11, which contains the `Timestamp` component.

```suggestion
| 4.11.x | `@openshift-console/dynamic-plugin-sdk` | 0.0.12 |
```

cc @rawagner 
/hold

Reply:
That is correct